test/user-agent-via-unpkg.test.ts

```diff
@@ -0,0 +1,16 @@
+import pkg from "../package.json";
+
+beforeAll(() => {
+  const script = document.createElement("script");
+  script.src = "https://unpkg.com/@sajari/sdk-js@2.0.0-alpha.18/dist/index.js";
```

Comment:
Can we see if there's a way we can get the version somehow? 

To give some background, the issue we're trying to tackle is when folks load it like this:

```html
<script src="https://unpkg.com/@sajari/sdk-js"></script>
```
(which will redirect to `https://unpkg.com/@sajari/sdk-js@1.0.10/dist.umd/index.js`) 

Which loads the `latest` tagged version, currently `1.x`. If we push `2.x` as `latest`, then it'll break for folks loading as above. Not great. 

What I was going to see is if there was any way we'd see `https://unpkg.com/@sajari/sdk-js` rather than `https://unpkg.com/@sajari/sdk-js@1.0.10/dist.umd/index.js` for the `document.currentScript.src` value. If we could get that info, it might be useful to narrow down who's most likely to break and get them to update to following:

```html
<script src="https://unpkg.com/@sajari/sdk-js@1"></script>
```

Reply:
I just published a dummy package on npm (`cdn-test-tuanddd`) that will run `console.log(document.currentScript)` on load, you can try it out and see.